### ISSUE TITLE
Support for graphql v0.10.x

### ISSUE THREAD

**Issue by dmk255** [bencooling/serverless-plugin-graphiql#5]
Thanks for making this plugin!

~Are there any plans to continue to support the newer versions of graphql?~

Didn't realize at first that the graphql dependency was for your example and not a dependency for the plugin.

I'd recommend having a separate `package.json` in the `/example` directory to keep dependencies separate. When I copied the example directly I was getting the following error:
```
 Error --------------------------------------------------

  Cannot find module 'graphql'
```

**Comment by bencooling** [comment]
@dmk255 I'm open to your suggestion, just wondering if having multiple package.json files in a repo a standard practice? Do you know of any prominent projects that do this? Is improved documentation in the README a better solution? Also open to pull requests if you would like to contribute!

**Comment by dmk255** [comment]
@bencooling Thanks for the quick response!

[apollographql/react-apollo](https://github.com/apollographql/react-apollo) is the first prominent repo that comes to mind.

The most frequent implementation I've seen is simply not providing the package dependencies at all ([AllenFang/react-bootstrap-table](https://github.com/AllenFang/react-bootstrap-table)). This is useful as a code reference but isn't that helpful to get started quickly in my opinion.

Documentation could be part of it. Personally, I'm a fan of a fully encapsulated example. I'm happy to submit a PR which removes `graphql` from the root package and moves that dependency to a new `package.json` in the example folder.

**Comment by bencooling** [comment]
@dmk255 Mate that sounds perfect! Would love for you to submit a PR.

**Comment by dmk255** [comment]
@bencooling Sounds good! It's PR https://github.com/bencooling/serverless-plugin-graphiql/pull/6

**Comment by bencooling** [comment]
Excellent. Thank you again @dmk255. Closed via #6.
